index.html

```diff
@@ -33,37 +33,49 @@
 <!-- end menu-->
 
 <!-- Include banner -->
-<div id="banner">
-    <div id="banner-text"></div>
+<div class="row">
+    <div class="col-4 pe-0 banner-text-container">
+        <div class="d-flex align-content-end justify-content-center flex-wrap banner-title">
```

Comment:
Does the weird spacing here affect the layout?

Reply:
Yes.  Normal vertical aligning wasn't working.

Reply:
I meant more how the code was aligned with the indentations.

I guess the actual question is: Is this how you wanted it? I don't love the way the two lines move when you resize the frame.

Also, the background colour is definitely different than the background for the globe. Does it look the same on your screen?

Reply:
I never noticed that the text moves.  I'll do something about that.

The background colour looked the same at home.  Looking at it now it looks imperceptibly different.  I have to look at it from different angles.  But if it's showing different on your screen it'll look different on other screens.  I'll look into it.

Reply:
Do you have the actual background colour? Because we should just set both to the same value right? Or is it more complicated than that?

Reply:
If the source code for the package doesn't tell you what the default background colour is or you can't figure it out another way, then just set your own background so we're in control of the colour